src/main/java/jp/ikedam/jenkins/plugins/extensible_choice_parameter/ChoiceListProvider.java

```diff
@@ -43,8 +44,10 @@
  *    <li>annotate the DescriptorImpl with Extension</li>
  * </ol>
  */
-abstract public class ChoiceListProvider extends AbstractDescribableImpl<ChoiceListProvider> implements ExtensionPoint
+abstract public class ChoiceListProvider extends AbstractDescribableImpl<ChoiceListProvider> implements ExtensionPoint, Serializable

```

Comment:
`ChoiceListProvider` should be `Serializable` as containing class `ExtensibleChoiceParameterDefinition` is `Serializable` (as its base class `ParameterDefinition` is `Serializable`).
I'm not sure why `ParameterDefinition` is `Serializable`...